File: axvoice.android/src/com/axia/VoiceMessage.java
package com.axia;

public class VoiceMessage {
	public static final int MT_RECORD_MSG = 1;
	public static final int MT_UPLOAD_MSG = 2;
	public static final int MT_DOWNLOAD_MSG = 3;
	public static final int MT_PLAY_MSG = 4;
	
	public long voiceID;
	public int msgType;
	public String[] params;
	
	public VoiceMessage(long voiceID, int msgType, String[] params) {
		this.voiceID = voiceID;
		this.msgType = msgType;
		this.params = params;
	}
}

File: axvoice.ios/axvoice.ios/VoiceMessage.m
//
//  VoiceMessage.m
//  axvoice.ios
//
//  Created by JinChao on 14/12/26.
//  Copyright (c) 2014年 thecodeway. All rights reserved.
//

#import <Foundation/Foundation.h>
#import "VoiceMessage.h"

@interface VoiceMessage()
{
    uint _voiceID;
    MessageType _msgType;
    NSArray* _params;
}

@end

@implementation VoiceMessage

@synthesize voiceID = _voiceID;
@synthesize msgType = _msgType;
@synthesize params = _params;


- (id)initWithParam : (uint)voiceID
            msgType : (MessageType)msgType
              params: (NSArray*)params
{
    if (self = [super init]) {
        
        _voiceID = voiceID;
        _msgType = msgType;
        _params = params;
    }
    return self;
}

@end
